minecraft_servers/kyriacity/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "server_name": "kyriacity",
+  "nice_name": "KyriaCity",
+  "direct_ip": "kyriacity.net",
+  "supported_languages": [
+     "en", "de"
+  ],
+  "social": {
+    "web": "https://kyriacity.net",
+    "web_shop": "https://shop.kyriacity.net",
```

Comment:
Please remove as long as shop is not available.

Reply:
done